index.html

```diff
@@ -125,6 +129,297 @@ <h1>I'm Moise Glad to see you !</h1>
 					</li>
 				</ul>
 			</section>
+			<section class="works">
+				<div class="card-work">
```

Comment:
- [ ] **Please `ul` to group all the projects and use the `li` tag for each project**. 

What this means is that you should use the `ul` tag to group all the projects together in one element. Then, inside of that ul element, you should use the `li` tag to display each project and its contents.

Why is it important?

Whenever you are trying to group and list things, you should consider using `ul` and `li` tags. HTML has the ability to express lists of things, and it helps the Google robot, screen readers, and all manner of users who don't care solely about the presentation of the site understand your content better.

Reply:
This issue is addressed! :clap: